lib/workers/repository/index.ts

```diff
@@ -92,3 +100,12 @@ export async function renovateRepository(
   logger.info({ durationMs: splits.total }, 'Repository finished');
   return repoResult;
 }
+
+// istanbul ignore next: renovateRepository is ignored
```

Comment:
do we really need the ignore? we should add tests instead

Reply:
Problem is that the entire repo worker function is being Istanbul ignored which is where this function is used.
I actually tried un-ignoring the repo worker function but it seemed out of scope for this PR as it requires quite a bit of work, maybe in another unrelated PR?

https://github.com/renovatebot/renovate/blob/f4bbf71b0b1d1cc7ea10ea922ed290bcc9ec5b75/lib/workers/repository/index.ts#L22-L23

Reply:
ok, let's do it in separate PR

Reply:
- https://github.com/renovatebot/renovate/issues/17993